dff/core/engine/conditions.py

```diff
@@ -39,18 +40,15 @@ def exact_match_condition_handler(ctx: Context, actor: Actor, *args, **kwargs) -
 
 @validate_arguments
 def regexp(pattern: Union[str, Pattern], flags: Union[int, re.RegexFlag] = 0, *args, **kwargs) -> Callable:
```

Comment:
Should we add Callable signature (☝️): `Callable[[Context, Actor, Any, Any], bool]` (AFAIK) here.